src/IW/Db/Repo.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+-- | SQL queries to work with the @issues@ table.
+
+module IW.Db.Repo
+       ( getRepos
+       , upsertRepos
+       ) where
+
+import IW.Core.Repo (Repo (..))
+import IW.Db.Functions (WithDb, executeMany, queryRaw)
+
+
+-- | Returns all repos in the database
+getRepos :: (WithDb env m) => m [Repo]
+getRepos = queryRaw [sql|
+    SELECT id, owner, name, descr, categories
+    FROM repos
```

Comment:
Let's also add `LIMIT 100` to `getRepos` and `getIssues`. Usually you never want to return everything from the table if it's big enough. Because it will be too expensive. So the solution is to give an ability to specif limit and offset. `LIMIT` always will be some hardcoded value, and  we will add `OFFSET` once we implement pagination.